core/src/test/scala/com/salesforce/op/stages/impl/insights/RecordInsightsLOCOTest.scala

```diff
@@ -404,10 +411,8 @@ class RecordInsightsLOCOTest extends FlatSpec with TestSparkContext with RecordI
    *
    * @param predicate  predicate used by RecordInsights in order to aggregate
    */
-  private def assertAggregatedWithPredicate(
-    predicate: OpVectorColumnHistory => Boolean,
-    testData: RecordInsightsTestData[LogisticRegressionModel]
-  ): Unit = {
+  private def assertAggregatedWithPredicate(predicate: OpVectorColumnHistory => Boolean,
+                                            testData: RecordInsightsTestData[LogisticRegressionModel]): Unit = {
```

Comment:
Can you revert the formatting here to the way it was before to match scala style guidelines?